service/routes.py

```diff
@@ -174,6 +176,40 @@ def update_inventory_records(product_id):
     existing_record.update(new_record)
     return jsonify(existing_record.serialize()), status.HTTP_200_OK
 
+
+@app.route("/inventory/checkout/<int:product_id>/<condition>/", methods=["PUT"])
+@app.route("/inventory/checkout/<int:product_id>/<condition>", methods=["PUT"])
+def checkout_quantity(product_id, condition):
+    """Reduces quantity from inventory of a particular item based on the amount specified by user"""
+    app.logger.info("Reduce quantity of item based on user requirement")
+    data = request.get_json()
+    app.logger.info(type(condition))
```

Comment:
either add text to logging or delete the line, this might not help in debugging.

Reply:
regarding line 186